rebuild-sites-amplify/change-redirs.js

```diff
@@ -0,0 +1,74 @@
+const {
+  AmplifyClient,
+  ListAppsCommand,
+  UpdateAppCommand,
+} = require("@aws-sdk/client-amplify");
+
+const MAX_RESULTS_PER_PAGE = 100
+
+const awsClient = new AmplifyClient({
+  region: "ap-southeast-1",
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+});
+
+async function changeRedirs() {
+  let sitesList = {}
+  let nextToken
+  const listCommand = new ListAppsCommand({
+    maxResults: MAX_RESULTS_PER_PAGE,
+  })
+  response = await (awsClient.send(listCommand))
+  nextToken = response.nextToken
+  while (nextToken) {
+    response.apps.forEach(app => {
+      if (!app.name) return
+      sitesList[app.name] = {
+        appId: app.appId,
+        branchName: 'master',
+        jobType: 'RELEASE',
+        customRules: app.customRules
+      }
+    })
+    const listCommand = new ListAppsCommand({
+      maxResults: MAX_RESULTS_PER_PAGE,
+      nextToken
+    })
+    response = await (awsClient.send(listCommand))
+    nextToken = response.nextToken
+  }
+  for (const site in sitesList) {
+    const siteInfo = sitesList[site]
+    let alreadyHasRule = false
+    siteInfo.customRules.forEach(rule => {
+      if (rule.source === "</%5c/>") alreadyHasRule = true
+      else if (rule.source === "</%5C/>") alreadyHasRule = true
```

Comment:
supernit: actually can just `.toLowerCase` but this is clear too

Reply:
also we can use `.some` here

Reply:
Swapped to check only for lowercase https://github.com/isomerpages/isomer-tooling/pull/44/commits/446ebd92845fb56a9f69b0cdb31fa412f68dee5d